### PR TITLE
added files required for Zuul-CI integration

### PR DESCRIPTION
**Pull request by goern**
This adds some base config for Coala and a basic Zuul-CI pipeline configuration. Bonus: a nice .gitignore file.

Signed-off-by: Christoph Görn <goern@redhat.com>

### CODE REVIEW

**Comment by goern** [comment]
Please keep in mind that the Zuul-CI is a Red Hat internal system, therefore the log files are only available internally.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.vscode
+amun/__pycache__/
```
Comment (durandom) [inline .gitignore]: this is thoth specific?

Comment (fridex) [reply .gitignore]: yes

Comment (fridex) [reply .gitignore]: so just `__pycache__` should work (all the `__pycache__` directories, which makes more sense anyway).

**Comment by goern** [comment]
BTW, we can also configure a 'gate' pipeline, that will merge based on passed checks and if a human approved the PR... so that humans dont have to merge stuff... or automated dependency updates opened by Kebechet could be auto-merge by Zuul

**Pull request merged**